packages/bddeb

```diff
@@ -100,8 +100,8 @@ def write_debian_folder(root, templ_data, cloud_util_deps):
     requires.extend(['python3'] + reqs + test_reqs)
     if templ_data['debian_release'] == 'xenial':
         requires.append('python3-pytest-catchlog')
-    elif templ_data['debian_release'] == 'impish':
-        requires.remove('dh-systemd')
+    elif templ_data['debian_release'] in ('xenial', 'bionic', 'focal'):
```

Comment:
Let's include "stretch" & "buster" so this still works on Debian-based images since you have removed dh-systemd from the common "debian" key in pkg-deps.json. I tested w/ LXC and bullseye and later don't have dh-systemd.

```suggestion
    elif templ_data['debian_release'] in ('stretch', 'buster', 'xenial', 'bionic', 'focal'):
```

Reply:
No need for stretch as it's python3.5 and cloud-init tip of main [doesn't support python3.5 anymore anyway](https://lists.launchpad.net/cloud-init/msg00377.html)